lib/haproxy_sock.js

```diff
@@ -39,6 +39,9 @@ function HaproxyCmdFSM(opts) {
     mod_assert.object(opts.log, 'opts.log');
     this.hcf_log = opts.log;
 
+    this.hcf_sockpath = process.env.TESTING === '1' ?
```

Comment:
Unless "TESTING" is already a standard of some sort, I'd prefer this be namespaced to something like "MUPPET_TESTING" or "MANTA_LOADBALANCER_TESTING". Sneaky generic envvars can be surprising (the `NODE_DEBUG=1` thing is somewhat of an example of that).

Also, could perhaps do this switching up where `HAPROXY_SOCK_PATH` is set. Then that commetn about keeping it in sync with `{etc,test}/haproxy.cfg.{in,test}` could apply.